packages/serverless/src/backend_connector.js

```diff
@@ -107,6 +107,15 @@ exports.setLogger = function setLogger(_logger) {
   logger = _logger;
 };
 
+/**
+ *
+ * "finalLambdaRequest":
```

Comment:
Please double check if my conclusion is correct. It looks to me the variable is at the moment mis-named, but if we have a proper comment it might be less confusing.

Reply:
@aryamohanan Did you make the same conclusion?

Reply:
In my opinion, your comments are accurate and clarify the situation well.